server/.openapi-generator-ignore

```diff
@@ -21,3 +21,5 @@
 #docs/*.md
 # Then explicitly reverse the ignore rule for a single file:
 #!docs/README.md
+
+physical_address_controller.py
```

Comment:
This means to ignore file the file `physical_address_controller.py` in this folder. We could one days discuss of the protocol for updating a server implementation. I now have a better understand of how this works since I did it many times last weekend. For now please remove the line.